src/cascade/input_data/configuration/construct_country.py

```diff
@@ -202,14 +202,12 @@ def convert_gbd_ids_to_dismod_values(with_ids_df, age_groups_df):
         pd.DataFrame: New pd.DataFrame with four added columns and in the same
             order as the input dataset.
     """
-    sex_df = pd.DataFrame(dict(x_sex=[-0.5, 0, 0.5], sex_id=[2, 3, 1]))
```

Comment:
Aw, I was trying to see how many places we could put this information. Figure we were up to 4 or 5.

Reply:
Sorry to spoil your fun